pmm/pmm2-upgrade-tests.groovy

```diff
@@ -199,39 +195,38 @@ pipeline {
         always {
             // stop staging
             sh '''
-                curl --insecure ${PMM_URL}/logs.zip --output logs.zip
+                curl --insecure ${PMM_URL}/logs.zip --output logs.zip || true
             '''
             destroyStaging(VM_NAME)
             sh '''
-                sudo chmod 777 -R pmm-app/tests/output
-                ./pmm-app/node_modules/.bin/mochawesome-merge pmm-app/tests/output/parallel_chunk*/*.json > pmm-app/tests/output/combine_results.json
-                ./pmm-app/node_modules/.bin/marge pmm-app/tests/output/combine_results.json --reportDir pmm-app/tests/output/ --inline --cdn --charts
+                sudo chmod 777 -R tests/output || true
```

Comment:
I don't think we need this line anymore, because we don't use the docker container for running tests anymore, could you remove this and try testing it on any one of the pipeline.

Reply:
should I try removing it in other pipelines?